apps/condo/domains/property/components/panels/Builder/BuildingPanelEdit.tsx

```diff
@@ -206,14 +213,27 @@ export const BuildingPanelEdit: React.FC<IBuildingPanelEditProps> = (props) => {
     const AllSectionsTitle = intl.formatMessage({ id: 'pages.condo.property.SectionSelect.AllTitle' })
     const SectionPrefixTitle = intl.formatMessage({ id: 'pages.condo.property.SectionSelect.OptionPrefix' })
 
-    const { mapValidationError, map, updateMap: updateFormField, handleSave, address } = props
+    const { mapValidationError, map, updateMap: updateFormField, handleSave, property } = props
+
+    const quickSave = Property.useUpdate({}, () => notification.success({
+        message: ChangesSaved,
+        placement: 'bottomRight',
+    }))
+    const debouncedQuickSave = useCallback(debounce(() => quickSave({ map }, property), 800), [map, property])
+    const quickSaveCallback = useCallback((event) => {
+        event.preventDefault()
+        debouncedQuickSave()
+    }, [debouncedQuickSave])
+
+    useHotkeys('ctrl+s', quickSaveCallback, [map, property])
```

Comment:
what will happen in case of creating house without setting required fields?

Reply:
Added test case for this & show error notification with description

Reply:
cool 👍